letsencrypt/translations/en.yaml

```diff
@@ -29,6 +29,12 @@ configuration:
       By default, The addon uses Let's Encrypt's default server at
       https://acme-v02.api.letsencrypt.org/. You can instruct the addon to use a
       different ACME server.
+  chain:
+    name: Preferred Certificate Chain
+    description: >-
+      If the CA offers multiple certificate chains, prefer the chain whose
+      topmost certificate was issued from this Subject Common Name. If no match,
+      the default offered chain will be used.
```

Comment:
```suggestion
      If you are using a much older Android device you may need to change this
      to `DST Root CA X3` to prevent certificate errors. Otherwise you should
      leave this option at `ISRG Root X1`.
```